app/javascript/src/ChangePassword/components/PasswordInput.jsx

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import React from 'react'
+import { PasswordField, validateSingleField } from 'LoginPage'
+
+type Props = {
+  isRequired?: boolean,
+  name: string,
+  label: string,
+  value: string,
+  isValid: boolean,
+  setValue: (value: string) => void,
+  setIsValid: (isValid: boolean) => void,
+  autoFocus?: string
+}
+
+const PasswordInput = ({ isRequired, name, label, value, isValid, setValue, setIsValid, autoFocus }: Props) => {
+  const inputProps = {
+    isRequired,
+    name: `user[${name}]`,
+    fieldId: `user_${name}`,
+    label,
+    value,
+    isValid,
+    onChange: (value, event) => {
+      const isValid = validateSingleField(event)
```

Comment:
I know this is not from this PR, but it's weird having a fn `validateSingleField` that takes an `event` as argument instead of the field

Reply:
I'd address that issue as a separate task, as it will affect all other forms